--- origsrc/qtbase-opensource-src-5.1.1/mkspecs/features/qt_tool.prf	2013-08-25 13:03:47.000000000 -0500
+++ src/qtbase-opensource-src-5.1.1/mkspecs/features/qt_tool.prf	2014-01-02 23:48:17.545298900 -0600
@@ -43,3 +43,6 @@ CONFIG += console
         cache(QT_TOOL.$${MODULE}.$$var, transient)
 
 }
+
+contains(QT_CONFIG, system-zlib): LIBS_PRIVATE += -lz
+host_build: LIBS += -L$$[QT_HOST_LIBS]
--- origsrc/qtbase-opensource-src-5.1.1/src/tools/bootstrap/bootstrap.pro	2014-01-01 13:22:42.367935800 -0600
+++ src/qtbase-opensource-src-5.1.1/src/tools/bootstrap/bootstrap.pro	2014-01-02 23:38:20.707687700 -0600
@@ -148,7 +148,7 @@ macx {
 
 *-g++*: QMAKE_CXXFLAGS += -ffunction-sections
 
-if(contains(QT_CONFIG, zlib)|cross_compile):include(../../3rdparty/zlib.pri)
+if(contains(QT_CONFIG, zlib)):include(../../3rdparty/zlib.pri)
 else:include(../../3rdparty/zlib_dependency.pri)
 
 win32:LIBS += -luser32 -lole32 -ladvapi32
