--- origsrc/qtbase-opensource-src-5.1.1/qmake/generators/win32/mingw_make.cpp	2013-08-25 13:04:06.000000000 -0500
+++ src/qtbase-opensource-src-5.1.1/qmake/generators/win32/mingw_make.cpp	2014-01-02 01:24:31.640046700 -0600
@@ -71,7 +71,7 @@ QString MingwMakefileGenerator::escapeDe
 
 QString MingwMakefileGenerator::getLibTarget()
 {
-    return QString("lib" + project->first("TARGET") + project->first("TARGET_VERSION_EXT") + ".a");
+    return QString("lib" + project->first("TARGET") + project->first("TARGET_VERSION_EXT") + ".dll.a");
 }
 
 bool MingwMakefileGenerator::findLibraries()
@@ -274,7 +274,7 @@ void MingwMakefileGenerator::init()
         if(!project->first("DESTDIR").isEmpty())
             destDir = Option::fixPathToTargetOS(project->first("DESTDIR") + Option::dir_sep, false, false);
         project->values("MINGW_IMPORT_LIB").prepend(destDir + "lib" + project->first("TARGET")
-                                                         + project->first("TARGET_VERSION_EXT") + ".a");
+                                                         + project->first("TARGET_VERSION_EXT") + ".dll.a");
 	project->values("QMAKE_LFLAGS").append(QString("-Wl,--out-implib,") + project->first("MINGW_IMPORT_LIB"));
     }
 
